apply_diff.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+# usage `./apply_diff.sh 500`
+MODULE_COUNT=$1
+declare -A MAX_MEMORY=(
+  ["50"]="400m"
+  ["100"]="600m"
+  ["200"]="1300m"
+  ["500"]="4000m"
+  ["1000"]="9000m"
+  ["app"]="20000m"
+  ["all"]="60g"
```

Comment:
I haven't tried either of these with the latest nightly, so let's just start here and see

Reply:
Ack.